app/assets/stylesheets/recipe.css.scss

```diff
@@ -0,0 +1,192 @@
+$color-light-yellow: #FFF888;
+$color-dirt-yellow: #E8AC06;
+$color-orange: #FF8508;
+$color-red: #E83505;
+$color-raisin: #5C0B22;
+$texture-picnic-pattern: linear-gradient(
+      rgba(0, 0, 0, 0.45), 
+      rgba(0, 0, 0, 0.45)
+    ), image-url('lunch_1x.jpg');
+$texture-food-pattern: linear-gradient(
+      rgba(0, 0, 0, 0.45), 
+      rgba(0, 0, 0, 0.45)
+), image-url('food.png');
+$texture-marble-pattern: linear-gradient(
+      rgba(0.2, 0.2, 0.2, 0.45), 
+      rgba(0.2, 0.2, 0.2, 0.45)
+), image-url('congruent_outline.png');
+$texture-wood-pattern: linear-gradient(
+      rgba(0.2, 0.2, 0.2, 0.1), 
+      rgba(0.2, 0.2, 0.2, 0.1)
+), image-url('retina_wood.png');
+$texture-notebook-pattern: linear-gradient(
+      rgba(0.1, 0.1, 0.1, 0.1), 
+      rgba(0.1, 0.1, 0.1, 0)
+), image-url('notebook_@2X.png');
+
+// $font4: 'Yellowtail', cursive;
+// $font4: 'Bad Script', cursive;
+// $font4: 'Marck Script', cursive;
+$font-items: 'Bitter', serif;
+// $font4: 'Lobster', cursive;
+// $font4: 'Special Elite', cursive;
+$font-title: 'Satisfy', cursive;
+// $font4: 'Damion', cursive;
+$font-footer: 'Architects Daughter', cursive;
+// $font4: 'Cookie', cursive;
+
+.recipe-body {
+  // background: white;
+  background: $texture-wood-pattern;
+  // height: 580px;
+  border-radius: 10px;
+  padding: 20px;
+  box-shadow: 8px 8px 25px #111;
+}
+
+.recipe-button-row {
+  margin: 0 0 20px 0;
+}
+
+.recipe-back-button {
+  display: block;
+  box-shadow: 2px 2px 5px #111;
+}
+
+div.recipe-profile {
+  // background: image-url('recipedb-default-img.png') no-repeat;
+  // text-align: center;
+  background: transparent;
+  // border: 2px solid black;
+  // margin: 0;
+  // padding: 0;
+  display: inline-block;
+  // margin: 10px 0 0 0;
+}
+
+.recipe-image {
+  width: 100%;
+  height: 100%;
+  // padding: 10px;
+  // position: middle;
+}
+
+.recipe-info {
+  // border: 1px dotted black;
+  display: inline-block;
+  // background: $color-light-yellow;
+  border-radius: 10px;
+  background: $texture-notebook-pattern;
+  // margin: 0 10px 0 0;
+}
+
+.recipe-div1 {
```

Comment:
* A lot of refactoring can be done with .recipe-div1 div2 div3 etc. (same rules are repeated 5 times)
* Could you envisage better class names, e.g. .recipe-title, .recipe-description, .recipe-ingredients etc.?